oshi-core/src/main/java/oshi/software/os/linux/LinuxNetworkParams.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Oshi (https://github.com/dblock/oshi)
+ *
+ * Copyright (c) 2010 - 2017 The Oshi Project Team
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Maintainers:
+ * dblock[at]dblock[dot]org
+ * widdis[at]gmail[dot]com
+ * enrico.bianchi[at]gmail[dot]com
+ *
+ * Contributors:
+ * https://github.com/dblock/oshi/graphs/contributors
+ */
+package oshi.software.os.linux;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import oshi.software.os.NetworkParams;
+import oshi.util.ExecutingCommand;
+import oshi.util.FileUtil;
+
+public class LinuxNetworkParams implements NetworkParams{
+
+    private static final long serialVersionUID = 1L;
+
+    private static final Logger LOG = LoggerFactory.getLogger(LinuxNetworkParams.class);
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getHostName() {
+        String hn = FileUtil.getStringFromFile("/proc/sys/kernel/hostname");
+        int dot = hn.indexOf('.');
+        if(dot == -1) {
+            return hn;
+        } else {
+            return hn.substring(0, dot);
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getDomainName() {
+        return ExecutingCommand.getFirstAnswer("dnsdomainname");
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String[] getDnsServers() {
+        List<String> resolv = FileUtil.readFile("/etc/resolv.conf");
+        String key = "nameserver";
+        int maxNameServer = 3;
+        List<String> servers = new ArrayList<>();
+        for(String line: resolv){
+            if(!line.startsWith(key)) continue;
+
+            String value = line.substring(key.length())
+                .replaceFirst("^[ \t]+", "");
+            if(value.length() == 0 ||
+                value.charAt(0) == '#' || value.charAt(0) == ';') continue;
+            String val = value.split("[ \t#;]", 2)[0];
+            servers.add(val);
+            if(servers.size() >= maxNameServer) break;
```

Comment:
Use brackets and separate lines.  If you're using Eclipse IDE, use its code style for "Sun Java Conventions".  Also change your "cleanup" style to remove unused imports that I see the codacy-bot has flagged.